test/vitest/healthcareProfessionalsStore.spec.ts

```diff
@@ -0,0 +1,116 @@
+import { setActivePinia, createPinia } from 'pinia'
+import { expect } from 'chai'
+import { useHealthcareProfessionalsStore } from '@/stores/healthcareProfessionalsStore'
+import { Insurance, Locale, Degree, Specialty, type HealthcareProfessional } from '~/typedefs/gqlTypes.js'
+
+describe('HealthCareProfessionalStore', () => {
+    beforeEach(() => {
+        setActivePinia(createPinia())
+    })
+
+    it('should initialize with default values', () => {
+        const healthcareProfessionalsStore = useHealthcareProfessionalsStore()
+
+        expect(healthcareProfessionalsStore.healthcareProfessionalsData).to.deep.equal([])
+        expect(healthcareProfessionalsStore.selectedHealthcareProfessionalId).to.equal('')
+        expect(healthcareProfessionalsStore.selectedHealthcareProfessionalData).to.equal(undefined)
+        expect(healthcareProfessionalsStore.removedHealthcareProfessionalNames).to.deep.equal([])
+        expect(healthcareProfessionalsStore.selectedFacilities).to.deep.equal([])
+        expect(healthcareProfessionalsStore.healthcareProfessionalSectionFields.names).to.deep.equal([])
+    })
+
+    it('should set selected healthcare professional and update the section fields', () => {
+        const healthcareProfessionalsStore = useHealthcareProfessionalsStore()
+        const mockHealthcareProfessional: HealthcareProfessional = {
+            id: '123',
+            acceptedInsurance: [Insurance.InsuranceNotAccepted],
+            createdDate: '2025-04-14',
+            updatedDate: '2025-04-15',
+            degrees: [Degree.Md],
+            facilityIds: ['facility1'],
+            names: [{ firstName: 'John', lastName: 'Doe', middleName: 'Deer', locale: Locale.EnUs }],
+            specialties: [Specialty.Neurology],
+            spokenLanguages: [Locale.EnUs]
+        }
+
+        healthcareProfessionalsStore.healthcareProfessionalsData = [mockHealthcareProfessional]
+        healthcareProfessionalsStore.setSelectedHealthcareProfessional('123')
```

Comment:
Let's confirm that `selectedHealthcareProfessionalId` is an empty string before `setSelectedHealthcareProfessional`  is called. 🙏🏾